app/models/video.rb

```diff
@@ -1,6 +1,13 @@
 class Video < ActiveRecord::Base
-  belongs_to :category
+  belongs_to :category 
   #validates :title, :presence => true
   #validates :description, presence: true
   validates_presence_of :title, :description
+
+  def self.search_by_title(search_term)
+    return [] if search_term.blank?
+    where("title LIKE ?", "%#{search_term}%")
+  end
```

Comment:
I think this would be better as a scope - http://guides.rubyonrails.org/active_record_querying.html#scopes